SashaSales/src/main/java/ua/artcode/util/IO.java

```diff
@@ -0,0 +1,45 @@
+package ua.artcode.util;
+
+import java.io.*;
+
+/**
+ * Created by work on 15.11.2016.
+ */
+public class IO {
+
+    public static final String PATH_FOR_PROPERTY_NAME = "src/main/resources/propertyName.txt";
```

Comment:
rename IO, must be more clear name

use classpath path, get reources from a classpath

IO.class.getResource...